src/config/mod.rs

```diff
@@ -173,6 +173,7 @@ create_config! {
          or they are left with trailing whitespaces";
     ignore: IgnoreList, IgnoreList::default(), false,
         "Skip formatting the specified files and directories";
+    ignore_missing_submod: bool, false, true, "Ignore missing submodule error";
```

Comment:
All new options are unstable by default. Can you please change this to:
```suggestion
    ignore_missing_submod: bool, false, false, "Ignore missing submodule error";
```